pyamg/strength.py

```diff
@@ -283,14 +334,14 @@ def symmetric_strength_of_connection(A, theta=0):
         # if theta == 0:
         #     return A
 
```

Comment:
Ambivalent on the name change here for Sx, Sp, Sj

Reply:
I do think if we are in the python interface where people will be looking more there is something to be said for names consistent with the actual scipy data structures.

Reply:
That makes sense, but I think it's also important to maintain consistency throughout PyAMG.  So for now, I would vote labeling it Sx, Sp, Sj, so the naming here is the same as in other places.  

I wouldn't be opposed to renaming all Sx, Sp, Sj to something more consistent with Scipy data structures, but I would vote to do that in a separate PR.